openapi.yaml

```diff
@@ -36,22 +36,24 @@ paths:
         - in: query
           name: firstSize
           type: integer
-          default: 0
-          minimum: 0
+          default: 1
+          minimum: 1
           description: >
-            The size of the tree that you wish to prove consistency from (0 means the beginning of the log)
-            Defaults to 0 if not specified
+            The size of the tree that you wish to prove consistency from (1 means the beginning of the log)
```

Comment:
Just curious, why is 1 the start rather than 0?

Reply:
https://github.com/google/trillian/blob/2053c7648b44d5de45863c3ad12550b511ad6a14/server/validate.go#L83 wants > 0 for both sizes

Reply:
Ah strange, thanks!